README.md

```diff
@@ -1,29 +1,86 @@
 # Deno
 
-A secure runtime for JavaScript and TypeScript.
+[![Build Status - Cirrus][]][Build status] [![Twitter handle][]][Twitter badge]
 
-[![Build Status](https://github.com/denoland/deno/workflows/ci/badge.svg?branch=master&event=push)](https://github.com/denoland/deno/actions)
 
-Deno aims to provide a productive and secure scripting environment for the
-modern programmer. It is built on top of V8, Rust, and TypeScript.
+<img align="right" src=https://deno.land/logo.svg height="150px">
 
-Please read the [introduction](https://deno.land/manual#introduction) for more
-specifics.
+Deno is a *simple*, *modern* and *secure* runtime for **JavaScript** and **TypeScript** that uses V8 and is built in Rust.
 
-[Website](https://deno.land/)
+### Features
 
-[Manual](https://deno.land/manual)
+- Secure by default. No file, network, or environment access, unless explicitly enabled.
+- Supports TypeScript out of the box.
+- Ships only a single executable file.
+- Built-in utilities like a dependency inspector (deno info) and a code formatter (deno fmt).
+- Set of reviewed standard modules that are guaranteed to work with [Deno](https://deno.land/std/)
 
-[Install](https://github.com/denoland/deno_install)
+### Install
 
-[API Reference](https://doc.deno.land)
+Shell (Mac, Linux):
 
-[Style Guide](https://github.com/denoland/deno/blob/master/docs/contributing/style_guide.md)
+```sh
+curl -fsSL https://deno.land/x/install/install.sh | sh
+```
 
-[Module Repository](https://deno.land/x/)
+PowerShell (Windows):
 
-[Releases](Releases.md)
+```powershell
+iwr https://deno.land/x/install/install.ps1 -useb | iex
+```
 
-[Chat](https://discord.gg/deno)
+[Homebrew](https://formulae.brew.sh/formula/deno) (Mac):
 
-[More links](https://github.com/denolib/awesome-deno)
+```sh
+brew install deno
+```
+
+[Chocolatey](https://chocolatey.org/packages/deno) (Windows):
+
+```powershell
+choco install deno
+```
+
+Build and install from source using [Cargo](https://crates.io/crates/deno):
+```sh
+cargo install deno
+```
+
+See  [deno_install](https://github.com/denoland/deno_install/blob/master/README.md) and [releases](https://github.com/denoland/deno/releases) for other options.
+
+### Getting Started
+
+Try running a simple program:
+
+```sh
+deno run https://deno.land/std/examples/welcome.ts
+```
+
+Or a more complex one:
+
+```sh
+import { serve } from "https://deno.land/std@0.69.0/http/server.ts";
+const s = serve({ port: 8000 });
+console.log("http://localhost:8000/");
+for await (const req of s) {
+  req.respond({ body: "Hello World\n" });
+}
+```
+
+You can find a more in depth introduction, examples, and environment setup guides in the [manual](https://deno.land/manual).
+
+More in-depth info can be found in the runtime [documentation](doc.deno.land)
+
+### Contributing
+
+We appreciate your help!
+
+To contribute, please read the our [guidelines](https://github.com/denoland/deno/blob/master/docs/contributing/style_guide.md)
```

Comment:
```suggestion
To contribute, please read the our [guidelines](https://deno.land/manual/contributing)
```

Reply:
I think we should use deno.land/manual instead of a markdown file

Reply:
```suggestion
To contribute, please read our [guidelines](https://deno.land/manual/contributing).
```